test/Libraries/AnalysisTests/AnalysisTests.cs

```diff
@@ -8,9 +8,9 @@
 
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.